src/OmniSharp.Script/ScriptProjectSystem.cs

```diff
@@ -43,7 +41,24 @@ public class ScriptProjectSystem : IProjectSystem
             _env = env;
             _logger = loggerFactory.CreateLogger<ScriptProjectSystem>();
             _projects = new Dictionary<string, ProjectInfo>();
-            _scriptProjectProvider = ScriptProjectProvider.Create(loggerFactory);
+
+            _compilationDependencyResolver = new CompilationDependencyResolver(type =>
+            {
+                // Prefix with "OmniSharp" so that we make it through the log filter.
+                var categoryName = $"OmniSharp.{type.FullName}";
```

Comment:
I think this maybe needs one more level nesting for better context i.e. `OmniSharp.Script`

Reply:
Done :)